lib/spec/openapi/utils.js

```diff
@@ -303,6 +303,17 @@ function resolveCommonParams (container, parameters, schema, ref, sharedSchemas,
   arr.forEach(swaggerSchema => parameters.push(swaggerSchema))
 }
 
+// Keys to references could be different than the name of the $ref,
+// for example `def-0`
+function findReference (definitions, id) {
+  const schemaKey = Object.keys(definitions).find(key => {
+    const schema = definitions[key]
+    return schema.$id === id
+  })
+
+  return definitions[schemaKey] ?? {}
+}
```

Comment:
Using `.values` makes sense actually, I'll change that 💪🏼 As for renaming the function, we could also go with that. My objective was having a function where you could resolve the reference for any other kinds of needs within this file for the future, but I can see how that's looking too far ahead and how it can be optimised.

Will make that change to return the description directly 👍🏼